CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+### Removed
+
+- Previously deprecated `/gov/jwt_keys/all` has been removed. The `/gov/service/jwk` endpoint to be used instead.
+
+### Changed
+
+- JWT authentication now supports raw public keys along with certificates (#6601).
+  - Public key information ('n' and 'e' claims) now have a priority if defined in JWK set, 'x5c' remains as a backup option.
```

Comment:
That's RSA only, do we support EC JWKs for OpenID?

Reply:
We don't. I'm very keen to know if anyone uses it, but it's listed "RECOMMENDED" to support, and all I've checked (Google, FB, MSFT) use `RS256`.

I think it's relatively cheap to add though, probably I shall partially return that, not sure if in this PR or after #6673.

Also, I'm curious how it shall be implemented, seems like constructing keys from `n` + `e` in RSA-specific.

I didn't manage to easily find an example or an explicit mentioning of how exactly it's passed in JWK, I'm guessing it may by via `x5c` claim

Anyway, I'll take a closer look at it before we decide

Reply:
Funny enough, the only MANDATORY to support is "HMAC-RSA", which I'm sure we don't do

Reply:
As discussed, we want to support EC and Ed too, but that can be separate PRs if desired.

Reply:
* EC support **added back**.
* EdDSA will be done/closed in #6713. More details there.